libraries/core_libs/network/src/tarcap/packets_handlers/common/ext_syncing_packet_handler.cpp

```diff
@@ -166,23 +166,12 @@ std::pair<bool, std::unordered_set<blk_hash_t>> ExtSyncingPacketHandler::checkDa
   expected_level = std::max(pivot_block->getLevel(), expected_level);
   expected_level++;
   if (expected_level != block.getLevel()) {
-    LOG(log_nf_) << "Invalid block level " << block.getLevel() << " for block " << block.getHash().abridged()
+    LOG(log_er_) << "Invalid block level " << block.getLevel() << " for block " << block.getHash().abridged()
```

Comment:
This one could be an error because only a malicious node would create invalid level

Reply:
I kept this one error, others were changed to trace